docs/docs-beta/docs/dagster-plus/getting-started.md

```diff
@@ -3,4 +3,63 @@ title: "Getting started with Dagster+"
 displayed_sidebar: "dagsterPlus"
 ---
 
-# Get started with Dagster+
\ No newline at end of file
+# Get started with Dagster+
+
+Dagster+ is a managed orchestration platform built on top of Dagster's open source engine.
+
+We run:
+ - Dagster's web UI at https://dagster.plus
+ - API servers at https://agent.dagster.plus
+ - Metadata stores for data cataloging and cost insights
+ - Backend services for orchestration, alerting, and more
+
+You run:
+ - Agents that listen to our API servers for new work
+ - The compute resource for  your data assets
+
+TODO: Architecture diagram?
+
+## Agents
+
+The Dagster+ agent is a long-lived process that polls Dagster+'s API servers for new work.
+
+To run an agent, first configure Dagster+ with an agent token:
+
+```
+dagster-plus configure
```

Comment:
Not `dagster-cloud`? Are there two CLIs?

Reply:
They're aliases for the same thing. I'm happy to put whichever one we think is more clear.

Reply:
I think since they need to pip install dagster-cloud, we should reference dagster-cloud as the CLI for now

Reply:
+1